yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,878 @@
+module vyatta-protocols-frr-isis-v1 {
```

Comment:
Please use this file instead: https://github.com/FRRouting/frr/blob/master/yang/frr-isisd.yang

It will help further integration of next FRR release